src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,48 @@
+.nav {
+  display: none;
+
+  @include tablet {
+    display: flex;
+  }
+
+  &__item {
+    position: relative;
+    margin: 0 10px;
+
+    &:first-child {
+      margin-left: 0;
+    }
+
+    &:last-child {
+      margin-right: 0;
+    }
```

Comment:
apply margin only for one side right or left 20px and no need to reset margin for first and last child, do like in file stars.scss on 11 line